### PR TITLE
AB#5022

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
This is working for me in local build, @DallasP9124 @r-joyce please review and merge.

**Comment by BigBallard** [comment]
@johnjdl did you test functionality? Just checking.

**Comment by johnjdl** [comment]
@DallasP9124 Yes, screenshot in ADO ticket. 
I was able to successfully create a new risk log.

**Comment by johnjdl** [comment]
@DallasP9124 @r-joyce please review and merge.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js [hunk 1]
```diff
@@ -146,27 +148,69 @@ class RiskTrackingPopover extends Component {
...
+        input: finalValues,
+      },
+      setSubmitting,
+      onCompleted: (data) => {
+        setSubmitting(false);
+        resetForm();
+        this.handleClose();
+        this.props.refreshQuery();
+      },
+      onError: (err) => console.log('RiskTrackingPopoverError', err),
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js]: @sachin755 please apply the toast here as well

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js [hunk 2]
```diff
@@ -207,17 +251,15 @@ class RiskTrackingPopover extends Component {
...
       R.assoc('entry_type', node?.entry_type || []),
       R.assoc('title', node?.name || ''),
       R.assoc('description', node?.description || ''),
       R.assoc('event_start', dateFormat(node?.event_start)),
       R.assoc('event_end', dateFormat(node?.event_end)),
-      R.assoc('logged_by', riskTrackingLoggedBy?.name || ''),
+      R.assoc('logged_by', riskTrackingLoggedBy?.name || []),
       R.assoc('status_change', node?.status_change || ''),
-      R.assoc('related_response', riskStatusResponse?.name || []),
+      R.assoc('related_responses', riskStatusResponse.map((value) => value.id) || []),
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:262]: @sachin755 correct me if I am wrong but if `riskStatusReponse` is empty, wont the result be an empty array already? Wouldn't it be appropriate to check if `riskStatusResponse` is not null instead? Just checking.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js [hunk 3]
```diff
@@ -487,12 +529,13 @@ class RiskTrackingPopover extends Component {
                           name="related_responses"
                           fullWidth={true}
                           size="small"
+                          multiple={true}
                           variant='outlined'
                           style={{ height: '38.09px' }}
                           containerstyle={{ width: '100%' }}
                         >
                           {riskStatusResponse.map((value, i) => (
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:537]: @sachin755 same here, is it necessary to check if `riskStatusResponse` is null or will it always at the least be an empty array?

**Review by BigBallard — CHANGES_REQUESTED** [review]

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js (hunk 3 above)
Comment (AMAN-SHARMA-82591) [reply opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:537]: @DallasP9124  it'll have an array of objects or an empty array.

**Review by r-joyce — APPROVED** [review]
Looks good to me 👍

**Pull request merged**